<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormIncluirTipodocumento</class>
 <widget class="QWidget" name="FormIncluirTipodocumento">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">text-transform: uppercase;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_2{

}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>700</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
border:4px solid blue;
	background-color: rgb(255, 255, 255);
}

QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
text-transform: uppercase;
}
QLineEdit:Focus {
border: 1px solid red;
}

QLabel{
text-transform: uppercase;
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="styleSheet">
            <string notr="true">QFrame#frame_6{
color:red;
border: none;
background-color:#2408D4;}

QLabel{color: white;
background-color:transparent;
font: 75 15pt &quot;MS Shell Dlg 2&quot;;
border:none
}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>tipo documento</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="styleSheet">
            <string notr="true">border:none;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_4">
              <property name="minimumSize">
               <size>
                <width>500</width>
                <height>150</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame{
background: #1E87F0;
border: 1px solid #1E87F0;
}
QLabel{
font-size: 16px;
font-family: &quot;Arial&quot;;
font-weight: bold;
color: white;
border: none
}
QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font-size: 16px;
font-family: &quot;Arial&quot;;
font-weight: bold;
text-transform: uppercase;

}
QLineEdit:Focus {
border: 1px solid red;
	background-color: rgb(255, 255, 255);r
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QLineEdit" name="tx_codigo">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>151</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="maxLength">
                <number>3</number>
               </property>
               <property name="placeholderText">
                <string>codigo</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="tx_descricao">
               <property name="geometry">
                <rect>
                 <x>180</x>
                 <y>30</y>
                 <width>391</width>
                 <height>41</height>
                </rect>
               </property>
              </widget>
              <widget class="QLabel" name="label_2">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>121</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>codigo</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_3">
               <property name="geometry">
                <rect>
                 <x>180</x>
                 <y>10</y>
                 <width>121</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>descriçao</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tab_Tipodocumento">
              <property name="focusPolicy">
               <enum>Qt::WheelFocus</enum>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::ActionsContextMenu</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QTableView{
color: black;
font-weight: bold;
font-size: 16px;
border:1px solid;
selection-color: rgb(0, 0, 255);
}
QHeaderView:section{
background: #FFF;

font-size: 13px;
font-family: &quot;Arial&quot;;
font-weight: bold;
color: #797979;
border:none;
border-bottom: 2px solid black;
border-top:2px solid black;
padding:5px 0px;
text-transform: uppercase
}</string>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="tabKeyNavigation">
               <bool>false</bool>
              </property>
              <property name="showDropIndicator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="dragDropOverwriteMode">
               <bool>false</bool>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>codigo</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>descriçao</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="bt_incluir">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>41</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: #1E87F0;
color: #FFF;
padding:5px 40px;
 }
QPushButton:hover{
background-color: #40a286
}</string>
              </property>
              <property name="text">
               <string>incluir</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bt_apagar">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>41</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: #7AB32E;
color: #FFF;
padding:10px 40px;
text-align:center;
 }
QPushButton:hover{
background-color: #40a286
}</string>
              </property>
              <property name="text">
               <string>apag ar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bt_cancelaIcluir">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>41</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: #7AB32E;
color: #FFF;
padding:5px 25px;
 }
QPushButton:hover{
background-color: #40a286
}</string>
              </property>
              <property name="text">
               <string>cancelar</string>
              </property>
              <property name="shortcut">
               <string>Esc</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="bt_grava_altera">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>41</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: #1E87F0;
color: #FFF;
padding:5px 40px;
 }
QPushButton:hover{
background-color: #40a286
}</string>
              </property>
              <property name="text">
               <string>gravar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bt_cancelar_Altera">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>41</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: #7AB32E;
color: #FFF;
padding:5px 40px;
 }
QPushButton:hover{
background-color: #40a286
}</string>
              </property>
              <property name="text">
               <string>cancelar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionaltera_documento">
   <property name="text">
    <string>altera documento</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
